src/PreviewCard/PreviewCard.test.js

```diff
@@ -1,33 +1,42 @@
 import React from "react";
 import {render, screen} from "@testing-library/react";
-import "@testing-library/jest-dom";
+import {Router, MemoryRouter} from "react-router-dom";
+import {history} from "history";
+import {createMemoryHistory} from "history";
 import userEvent from "@testing-library/user-event";
 import PreviewCard from "./PreviewCard";
+import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
 
 describe("Card", () => {
+  const fakeClick = jest.fn();
+  const historyMock = {push: fakeClick};
   beforeEach(() => {
-    const fakeClick = { push: jest.fn(id) }
- 
+    const history = createMemoryHistory();
     render(
-      <PreviewCard
-        average_rating={9}
-        backdrop_path="testback"
-        id={694919}
-        poster_path="testpath"
-        release_date="test-date"
-        title="Test Title"
-        onClick={fakeClick}
-      />
+      <Router history={history}>
+        <PreviewCard
+          average_rating={9}
+          backdrop_path="testback"
+          id={694919}
+          poster_path="testpath"
+          release_date="test-date"
+          title="Test Title"
+          history={history}
```

Comment:
Currently the best way we have found to replicate history.push. May need to get rid of it entirely if we can. Possibly a callback sent back to app?

Reply:
Yeah, like have a parent (class) component be able to re-render by using `setState()`/etc?